Api/src/main/java/com/dingdong/api/user/service/UserService.java

```diff
@@ -14,11 +19,11 @@ public class UserService {
 
     private final UserRepository userRepository;
 
+    @Deprecated(since = "DEP-63", forRemoval = true)
```

Comment:
반영 굿!
슬랙으로 잠깐 피드백 했지만 그래도 커밋상 기록에 남아있으면 좋을 것 같아서 기록합니당!

더 이상 로직이 필요 없어졌을때 주석으로 처리를 하는 경우에, 다른 사람들이 이유가 있어서 남겨놓았으리라 생각하고 지우는 것을 주저하는 경우가 많다고 합니다.  이게 쌓이다 보면 레거시가 되기도 하구용ㅎㅎ.. 
cc.. https://velog.io/@hangem422/clean-code-comment

그래서 지금 @sanbonai06 님이 걸어주신 것처럼 @Deprecated(since = {버전}, forRemoval = {true or false})를 걸어놓는 것을 권장하는데, 보통 저는 즉시 삭제해도 되는 경우는 forRemoval = true, 한번 고민해볼 문제는 false로 걸어두는 편입니다.!
@Deprecated 걸어놓으면 컴파일에서 경고 문구를 주기도 해서 리마인드 되기도 하니 저희도 주석 처리보다는 @Deprecated를 사용하는 편으로 합시당 @Sim-mi-gyeong

Reply:
너무 좋습니당 forRemoval로 의미 나누는건 처음 알아서 좋네용 감사합니당! 바로 머지할게요

Reply:
옹,, forRemoval 로 의미 나누는 거 저도 처음 알았어용.. 감사합니닷!-! 👻